RabbitMQ.Stream.Client/IConsumer.cs

```diff
@@ -40,4 +42,24 @@ public record IConsumerConfig : INamedEntity
     public string Reference { get; set; }
 
     public Func<string, Task> ConnectionClosedHandler { get; set; }
+
+    // InitialCredits is the initial credits to be used for the consumer.
+    // if the InitialCredits is not set, the default value will be 2.
+    // It is the number of the chunks that the consumer will receive at beginning.
+    // A high value can increase the throughput but can increase the memory usage and server-side CPU usage.
+    // The RawConsumer uses this value to create the Channel buffer so all the chunks will be stored in the buffer memory.
+    // The default value it is usually a good value.
+    public ushort InitialCredits
+    {
+        get => _initialCredits;
+        set
+        {
+            if (value < 1)
+            {
+                throw new ArgumentException("InitialCredits must be greater than 0. Default value is 2.");
```

Comment:
Maybe make this message dynamic and reference magic value? Might get out of sync in case you'll want to increase this in the future

Reply:
Done via https://github.com/rabbitmq/rabbitmq-stream-dotnet-client/pull/265/commits/10ebdec75c2293490740f4805ac09cd40711d212